usi/src/main/java/daq/usi/ovs/OpenVSwitch.java

```diff
@@ -31,7 +33,7 @@ protected String getInterfaceByPort(int devicePort) throws FileNotFoundException
       Matcher m = pattern.matcher(line);
       return m.find();
     }).findFirst().get();
-
+    bufferedReader.close();
```

Comment:
generall the java try-with-resources pattern is better than explicitly closing the object. https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html